include/tcframe/spec/core/StyleConfig.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <tuple>
+#include <utility>
+
+using std::move;
+using std::tie;
+
+namespace tcframe {
+
+struct StyleConfig {
+    friend class StyleConfigBuilder;
+
+private:
+    bool generateOutputFiles_;
+
+public:
+    bool generateOutputFiles() const {
+        return generateOutputFiles_;
+    }
+
+    bool operator==(const StyleConfig& o) const {
+        return generateOutputFiles_ == o.generateOutputFiles_;
```

Comment:
Put `tie()` here so that the next person that adds new fields doesn't have to do that (you imported `<tuple>` anyway